panacea-3/upgrade-validator.md

```diff
@@ -116,6 +116,32 @@ cat ~/genesis.40.json | \
   jq -c '.app_state.mint.params.inflation_max = "0.100000000000000000"' | \
```

Comment:
Could you also update the title of this section?

For example,

From:
```
7. Change validator-related parameters as discussed so far.
```
To:
```
7. Change validator-related parameters and new parameters for IBC and Wasm.
```

Reply:
OK I fixed it.

https://github.com/medibloc/panacea-launch/pull/8/commits/d72f207a9aaf28dc60fa0b605f906b34e798d454